doc/protocol/support.md

```diff
@@ -0,0 +1,134 @@
+# What do the KUKSA.val server and data broker support?
+This file contains a clear overview what the KUKSA.val server and data broker each supports. It focuses on gRPC and VISS support and also what feeders are supported.
+
+## Supported protocols
+
+
+| Protocol   |      Server      |  Broker   |
+|------------|:----------------:|----------:|
+| VISS V1    |      x           |     o     |

```

Comment:
Could we use the wording " KUKSA.val server supports only the semantic of VISS v1. The syntax is based on VISS v2" or is there a better naming because you quoted the word semantic and syntax.